src/tools/qt-gui/qml/GUIActions.qml

```diff
@@ -402,5 +408,16 @@ Item {
 		iconName: Helper.useIcon("configure")
 		onTriggered: appearanceSettingsWindow.show()
 	}
+
+	Action {
+		id: toggleViewerAction
+		checkable: true
+		checked: guiSettings.viewermode
+		onCheckedChanged: {
+			guiSettings.viewermode = checked
+			guiSettings.setKDB()
```

Comment:
Does the user expects this? Why not synchronize? Maybe its better to fail as long the user did not press synchronize before?